src/python/espressomd/particle_data.pyx

```diff
@@ -1678,6 +1678,62 @@ cdef class ParticleList:
                     if (p.type == t or t == "all"):
                         vtk.write("{} {} {}\n".format(*p.v))
 
+
+    def writevsf(self, fp, types='all'):
+        """
+        writes a VST (VTF Structure Format) to a file.
+        This can be used to write the header of a VTF file.
+
+        Parameters
+        ----------
+        types : str
+                Specifies the particle types. The string 'all' will write all particles
+        fp : file
+                   File pointer to write to.
+
+        """
+        global box_l
```

Comment:
What if the user doesn't name their box length variable `box_l`?  What if it is not an array?